csrc/scheduler/normalization.cpp

```diff
@@ -218,9 +212,9 @@ std::shared_ptr<ReductionParams> innerOuterPersistentHeuristic(
     rparams->tidx_for_outer_reduction = true;
     constexpr int64_t threads_per_block_mrpb = 512;
 
-    // Step-1, InnerParams, Reduction dim: inner_vect(reuse), inner_batch, bdimx
-    iop.inner_batch = 1;
-    iop.bdimx = inner_dim_numel / iop.inner_vect;
+    // Step-1, InnerParams, Reduction dim: inner_vect(reuse),
+    // inner_batch(reuse), bdimx
```

Comment:
What's the reason of not changing `inner_batch` to 1?

Reply:
changing inner_batch to 1 is a redundant operation. It was set in `getOptionalInnerOuterPersistentBufferBatches` and there is no need to change it again. Also, in this PR, `inner_batch` is no longer 1 if vectorization factor is 1. It is increased to set a minimum serial reduction workload for each thread.